src/Dialog/ConfirmationDialog.jsx

```diff
@@ -72,9 +79,16 @@ const ConfirmationDialog = ({
     cancelButtonText = 'Cancel',
     isOkButtonEnabled = true,
 }) => (
-    <Modal show={isVisible} onHide={onCancel} backdrop={isInProgress ? 'static' : false}>
+    <Modal
+        show={isVisible}
+        onHide={onCancel}
+        backdrop={isInProgress ? 'static' : false}
+        size="lg"
+    >
         <Modal.Header closeButton={!isInProgress}>
-            <Modal.Title data-testid="title">{title}</Modal.Title>
+            <Modal.Title data-testid="title">
+                <h3>{title}</h3>
```

Comment:
This change (and also the line https://github.com/NordicSemiconductor/pc-nrfconnect-shared/blob/85722f36bbccba24dff231e9347c04f5ff695a93/src/Dialog/ConfirmationDialog.jsx#L110) do change other existing dialogs. Is this ok?

E.g. the version problem dialog before looked like this:

![_Users_ms_Desktop_nRF%20Connect%203 3 0 app_Contents_Resources_app asar_resources_launcher html](https://user-images.githubusercontent.com/260705/89877930-1ffbfb00-dbc1-11ea-947f-aa210d265b4e.png)

and it is now changed to this:

![_Users_ms_w_pc-nrfconnect-launcher_resources_launcher html](https://user-images.githubusercontent.com/260705/89877955-28eccc80-dbc1-11ea-948c-f86d9104e9e6.png)

Reply:
It is ok. I mentioned it to Ketil and he agreed.